data/rest-data.go

```diff
@@ -226,17 +224,63 @@ func (r *RestData) getWorkflowFiles() error {
 	return err
 }
 
-func (r *RestData) foundSecurityInsights(content DirContents) string {
-	if strings.Contains(strings.ToLower(content.Name), "security-insights.") {
-		response, err := r.getSourceFile(r.owner, r.repo, content.Path)
+// returns true when a file with case insensitive name matching support.md is found in the root or forge directories or when the readme.md contains a heading named "Support"
+func (r *RestData) HasSupportMarkdown() bool {
+	if r.checkFile("support.md") != "" {
+		return true
+	}
+	readmePath := r.checkFile("readme.md")
+	if readmePath != "" {
+		contents, err := r.getSourceFile(r.owner, r.repo, readmePath)
+		if err != nil {
+			r.Config.Logger.Error(fmt.Sprintf("error getting readme contents: %s", err.Error()))
+			return false
+		}
+
+		headings := parseMarkdownHeadings(contents.ByteContent)
+		for _, heading := range headings {
+			if heading == "Support" {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func parseMarkdownHeadings(content []byte) []string {
+	var headings []string
+
+	// Parse markdown into AST
+	md := markdown.Parse(content, nil)
+
+	// Walk the AST and collect headings
+	ast.WalkFunc(md, func(node ast.Node, entering bool) ast.WalkStatus {
+		if heading, ok := node.(*ast.Heading); ok && entering {
+			// Get the text content of the heading
+			if len(heading.Children) > 0 {
+				if text, ok := heading.Children[0].(*ast.Text); ok {
+					headings = append(headings, string(text.Literal))
+				}
+			}
+		}
+		return ast.GoToNext
+	})
+
+	return headings
+}
+
+func (r *RestData) loadSecurityInsights() {
+	// TODO: si package should define a const for the expected file name
```

Comment:
https://github.com/ossf/si-tooling/pull/15

Reply:
```suggestion
```

sure, merged that in